p2p/3.IPFS_or_HTTP/SETUP.md

```diff
@@ -0,0 +1,38 @@
+# Setup
+
+In order to launch our music platform, you have to download a few things.
+* [Docker](./SETUP.md#dockerhttpswwwdockercom-fish)
+  * [What is Docker ?](./SETUP.md#what-is-dockerhttpswwwdockercom--question)
+  * [Installation on Ubuntu](./SETUP.md#installation-on-ubuntu)
+  * [Installation on Fedora](./SETUP.md#installation-on-fedorahttpsdocsdockercomengineinstallfedora)
+* [Docker-Compose](./SETUP.md#docker-compose)
+  * [Docker comp...what ?](./SETUP.md#docker-comp-what-)
+  * [Installation](./SETUP.md#installation)
+
+## [Docker](https://www.docker.com/) :fish:
+
+### What is [Docker](https://www.docker.com/) ? :question: 
+
+Docker enables developers to easily pack, ship, and run any application as a lightweight, portable, self-sufficient container,
+which can run virtually anywhere : that implies we can run our platform locally without any trouble.
+
+### Installation on Ubuntu
+I highly recommend you to use your official package manager.
+Otherwise, open a tab, and follow those [steps](https://docs.docker.com/engine/install/ubuntu/).
+
+### [Installation on Fedora](https://docs.docker.com/engine/install/fedora/)
```

Comment:
Use docker workshop setup for that part
As well, where can I get the code of your platform ? It should be in the setup